src/main/flight/gps_rescue.c

```diff
@@ -758,7 +778,8 @@ void gpsRescueUpdate(void)
             rescueState.intent.yawAttenuator += rescueState.sensor.gpsRescueTaskIntervalSeconds;
         }
         if (rescueState.sensor.absErrorAngle < 30.0f) {
-            rescueState.intent.pitchAngleLimitDeg = gpsRescueConfig()->maxRescueAngle; // allow pitch
+            // allow pitch, limiting allowed angle if there is a significant groundspeedPitchAttenuator
```

Comment:
```suggestion
            // allow pitch, limiting allowed angle if we are drifting away from home

Reply:
quad is leveled (roll,pitch=0) during previous phase. So groundspeed here is drift + wind; unless this phase was entered very quickly.
Value will be changed on next GPS update anyway ...

Reply:
Thank you, again your suggestion is excellent and the comment entirely correct.